docs/workflows/common-workflow-steps/custom-script.md

```diff
@@ -510,4 +511,76 @@ echo "Completed Build Log URL: $AC_COMPLETED_BUILD_LOG_URL"
 # To use these URL variables in later steps within the same workflow, add them to ENV_FILE_PATH as shown below; otherwise, they will not be accessible.
 echo "AC_COMPLETED_BUILD_LOG_URL=$AC_COMPLETED_BUILD_LOG_URL" >> $AC_ENV_FILE_PATH
 echo "AC_IN_PROGRESS_BUILD_LOG_URL=$AC_IN_PROGRESS_BUILD_LOG_URL" >> $AC_ENV_FILE_PATH
-```
\ No newline at end of file
+```
+
+### How do you store and use Custom Scripts in a Git repository for Custom Script steps??
+
+If you are looking for a modular way to manage your Appcircle CI custom scripts, you can host your script files in a Git repository and pull them dynamically into the build environment. This approach helps keep your workflow clean and allows centralized version control for your custom scripts.
+
+This section explains how to clone a private Git repository and execute a specific script file from it using the Custom Script step.
```

Comment:
Can you write a short tip box about what needs to be done when using a public repository?